apps/src/gamelab/GameLabP5.js

```diff
@@ -134,6 +245,56 @@ GameLabP5.prototype.init = function (options) {
       return s.animation ? s.animation.frameChanged : false;
     };
 
+    Object.defineProperty(s, 'x', {
```

Comment:
This section feels like a lot of copy-paste for something we might need more of.

Opposed to some variant of the code below, using lodash [_.get](https://lodash.com/docs#get) and [_.set](https://lodash.com/docs#set), to make it very easy to add aliases?

```javascript
[
  { alias: 'x', path: 'position.x' },
  { alias: 'y', path: 'position.y' },
  { alias: 'velocityX', path: 'velocity.x' },
  { alias: 'velocityY', path: 'velocity.y' },
  { alias: 'lifetime', path: 'life' }
].forEach(function (aliasEntry) {
  var alias = aliasEntry.alias,
      path = aliasEntry.path;
  Object.defineProperty(s, alias, {
    enumerable: true,
    get: _.get.bind(_, s, path),
    set: _.set.bind(_, s, path)
  });
});
```